packages/fluentui/react-northstar/src/components/Button/ButtonContent.tsx

```diff
@@ -32,9 +32,8 @@ ButtonContent.propTypes = {
   ...commonPropTypes.createCommon(),
   size: customPropTypes.size,
 };
-ButtonContent.create = createShorthandFactory({
-  Component: ButtonContent,
+ButtonContent.shorthandConfig = {
```

Comment:
any reason to not make this just a static `mappedProp`? is there anything else in `shorthandConfig`?